### ISSUE TITLE
Custom icons

### ISSUE THREAD

**Issue by tadaskarpavicius** [vlitejs/vlite#122]
### Describe the proposed feature.

Love this plugin, great job! 
However, one feature I missed greatly is the ability to supply my own svg icons. I know I can go around this in the css, but it is not the same as having svgs in the dom.

**Comment by yoriiis** [comment]
Hello @tadaskarpavicius, thanks for the compliment, I appreciate it!

About the issue, it is quite specific as a request. Do you want to change all the icons?

We could consider a SVG icon sprite skippable in the parameter, but that would involve replacing them all in the new sprite, it is not easy to use. Don't hesitate if you have an idea in mind

**Comment by tadaskarpavicius** [comment]
No, usually just some of them need to be replaced. 
In my case which prompted to create this ticket, I needed to replace only the Play button. It needed to change color on mouseover which meant that instead of manipulating fill css attribute of my custom icon I had to use two different color svg datauris in my css.

Not really sure what's the right approach.
Maybe player config option for custom icon svgs? 
That seems to be common on the carousel js plugins (a config prop for prev/next button html for example).

But then again, users could do this themselves on the onReady callback, so not sure if this needs to be included. 
Might go against the "lite" ethos.
